### ISSUE TITLE
Feature idea: Group subscriptions on one server with own headers

### ISSUE THREAD

**Issue by luebbe** [ottopaulsen/MMM-MQTT#33]
As far as I have understood the example it is right now possible to set a header for the entire module and labels for each subscription. Assuming we have several temp/humidity sensors on the same sever, it would be nice to subscribe to each of them and have a header for each.

Example:

**Indoor**
Temp 20°
Humid 56%
**Outdoor**
Temp -3°
Humid 34%

It could also be a good idea to define a kind of mqtt sensor template if similar sensors exist to avoid repeating configuration. This template could grab the subtopics underneath a base topic and render them.

**Comment by ottopaulsen** [comment]
Hi

I have no time for this module right now, as I am working on my MMM-Tibber module, but I will keep your request for later.

In the meantime you can try to set up the module twice. Then I think you can have different titles and different contents, and that way you can get the kind of grouping you want.

Repeating configuration is, in my opinion, not a big issue with the kid of powerful editors we have today, but I will keep it in mind.

Otto

**Comment by luebbe** [comment]
Always remember that nano or vi via SSH may be the tools of choice for configuration ;)

**Comment by ottopaulsen** [comment]
True. Dis you try to set up multiple module instances?

Otto

**Comment by luebbe** [comment]
Yes I did and it works fine. I wanted to have them next to each other in bottom_center, but unfortunately they stack. Have to find out how to achieve this. For now I put one in bottom_center and one in bottom_right, but if I want to show a third value, I have to find a solution.
